testing/ef_tests/tests/tests.rs

```diff
@@ -188,12 +189,21 @@ mod ssz_static {
     ssz_static_test!(checkpoint, Checkpoint);
     ssz_static_test!(deposit, Deposit);
     ssz_static_test!(deposit_data, DepositData);
+    ssz_static_test!(deposit_message, DepositMessage);
+    // FIXME(sproul): move Eth1Block to consensus/types
```

Comment:
Hello me! We could either do the move, or open an issue and leave a pointer here...

Reply:
I've created an issue: https://github.com/sigp/lighthouse/issues/1835

Reply:
Perfecto